client/src/app/site/pages/organization/modules/navigation/organization-navigation/organization-navigation.component.ts

```diff
@@ -17,13 +17,13 @@ export class OrganizationNavigationComponent {
         {
             route: `/`,
             displayName: `Calendar`,
-            icon: `event_available`,
+            icon: `auto_awesome_mosaic`,
```

Comment:
As discussed: Please use "apps" icon (as in meeting back button) instead of old committee icon "auto_awesome_mosaic".